File: Programming-Basics-with-Python/9.While Loop - Lab/03. Sum Numbers.py
start_number = int(input())
# повтаряме: четем число + сумираме
# спираме: сума >= start_number
# продължаваме: сума < start_number
sum = 0
while sum < start_number:
    number = int(input())
    sum += number
else: # sum >= start_number
    print(sum)

File: Programming-Basics-with-JavaScript/5.While Loop/03sumNumbers.js
function sumNumbers(input){
    let sumNums = 0
    let idx = 0
    
    let num = Number(input[idx])
    idx++

    let currentNum = Number(input[idx])
    idx++
    while (num > sumNums){
        sumNums += currentNum

        currentNum = Number(input[idx])
        idx++
    }

    console.log(sumNums)
    
}
sumNumbers(["100",
"10",
"20",
"30",
"40"])


